railties/lib/rails/generators/rails/plugin/templates/bin/test.tt

```diff
@@ -5,6 +5,6 @@ require 'rails/test_unit/minitest_plugin'
 
 Rails::TestUnitReporter.executable = 'bin/test'
 
-Minitest.run_via[:rails] = true
+Minitest.run_via = :rails
```

Comment:
Do we have any tasks akin to `rails app:update` for plugins that'll help updating this file and let us avoid the backwardscompatibility line?